ts/handler_utils/distributed/pt_pippy.py

```diff
@@ -0,0 +1,62 @@
+import importlib.util
+
+import os
+import time
+import torch
+from pkg_resources import packaging
+from ts.torch_handler.base_handler import BaseHandler
+import torch.distributed.rpc as rpc
+import pippy.fx
+from pippy import run_pippy
+from pippy.IR import MultiUseParameterConfig, Pipe
+from pippy.PipelineDriver import PipelineDriverFillDrain, PipelineDriver1F1B, PipelineDriverInterleaved1F1B, \
+    PipelineDriverBase
+from pippy.hf import PiPPyHFTracer
+from pippy.microbatch import TensorChunkSpec
+from pippy import split_on_size_threshold, split_into_equal_size
+import inspect
+
+def initialize_rpc_workers(local_rank,world_size):
+    options = rpc.TensorPipeRpcBackendOptions(
+        num_worker_threads=512,
+        rpc_timeout=1800
+    )
+    n_devs = torch.cuda.device_count()
+    dev_id = local_rank % n_devs 
+    for i in range (world_size):
+        options.set_device_map(f"worker{i}", {dev_id: i % n_devs})
+    print(
+        f"rank = {local_rank} pid/device = "
+        f"{os.getpid()}/{dev_id}"
+    )
+    rpc.init_rpc(f"worker{local_rank}",
+                    rank=local_rank,
+                    world_size=world_size,
+                    rpc_backend_options=options)
+
+
+def get_pipline_driver(model,world_size, input_names, model_type, chunks):
```

Comment:
letting the `get_pipline_driver` to get the world_size from env variable of torchrun fails here.

will need `input_names` otherwise we have to take the part of concrete_args --> which is based on input_name out of the function or split into two functions.

`model_type, chunks` model type is required as we need to know if its HF model to pass HF tracer to pippy_all_compile()

same goes to chunks.